### ISSUE TITLE
[qgroundcontrol] Current published version no longer builds

### ISSUE THREAD

**Issue by karcsesz** [acxz/pkgbuilds#254]
When attempting to build qgroundcontrol, I've encountered the following error:
```sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     47      0 --:--:-- --:--:-- --:--:--    47
ar: /[snip]/qgroundcontrol-4.2.6/build/airmap-platform-sdk.deb: file format not recognized
```

It seems that the repository that QGroundControl links to for AirMap has gone offline (alongside all of AirMap's respective operations).

QGroundControl has removed the dependency in version 4.2.8, and editing the package version does result in a successful build, so a bump would be much appreciated.

**Comment by fcladera** [comment]
Also fails in chroot.

**Comment by acxz** [comment]
Sorry for the delayed response. Fixed in 2d3ee71 and published to AUR.
